CRM/Event/Form/ManageEvent/TabHeader.php

```diff
@@ -44,6 +44,8 @@ class CRM_Event_Form_ManageEvent_TabHeader {
    * @return array
    */
   public static function build(&$form) {
+    $form->assign('selectedChild', CRM_Utils_Request::retrieve('selectedChild', 'String', $form));
```

Comment:
should this be alphanumeric or is that too restrictive?

Reply:
I think it probably could be, I'm not sure if there's any tabs with _ in the name

Reply:
@seamuslee001 I feel like we need another rule - Alphanumeric + a few more - hyphen, dash & possibly space

Reply:
In this case only 'user-profiles' is not Alphanumeric - & as a new assign we can be restrictive. I wonder if we can convert that to one word?

Reply:
Right, Alphanumeric includes the "-" and "_" characters so we're fine to switch this to alphanumeric - I've changed all four instances and updated the PR.